package.json

```diff
@@ -22,60 +22,52 @@
     "pretty": "pretty-quick",
     "pretty:staged": "pretty-quick --staged",
     "preview": "nuxt preview",
-    "test": "echo Fix this",
-    "test:watch": "echo Fix this",
-    "test-original": "vitest --run",
-    "test:watch-original": "vitest --watch"
+    "test": "vitest --run",
+    "test:watch": "vitest --watch"
   },
   "dependencies": {
     "@nuxtjs/i18n": "8.0.0-alpha.3",
-    "nuxt": "3.0.0-rc.11",
-    "unplugin-icons": "^0.14.10",
-    "unplugin-vue-components": "^0.22.8"
```

Comment:
Oh we're not using these anymore? What icon set are we using?

Reply:
I moved them to devDeps as they're used as plugins into viteConfig so don't need to be used at runtime